opm/common/OpmLog/EclipsePRTLog.hpp

```diff
@@ -29,16 +29,21 @@ namespace Opm {
 class EclipsePRTLog : public StreamLog {
 
 public:
-    using StreamLog::StreamLog;
 
     void addTaggedMessage(int64_t messageType, const std::string& messageTag, const std::string& message);
 
     size_t numMessages(int64_t messageType) const;
 
     ~EclipsePRTLog();
 
+    EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
As I am myself constantly overwhelmed be the verbosity of OPM's documentation ;) I will gladly add docs.